src/hooks/useLocalStorage.ts

```diff
@@ -26,6 +26,7 @@ export const useLocalStorage = (key: string, initialValue: any) => {
     const clearStorage = () => {
         try {
             window.localStorage.removeItem(key)
+            setStoredValue(initialValue)
```

Comment:
Thanks for taking a look through the code! :) Was bummed I could not get through as much as I'd hoped - it's a lot to try to fit into an hour. Maybe I'll be able to give this talk again in another venue after some revisions.

As for this particular function I was on the fence about this - for demonstration purposes I wanted to show that we could clear the value in localstorage without actually changing local state. Then upon refreshing the page or switching back and forth on pages we would see the current input value vanish. 

I think if I intended to use this hook in a real app though I would opt to go this route though.